modules/test/playwright/tests/object-web/utils/mockObjectFields.ts

```diff
@@ -147,6 +147,41 @@ function getFormatDate(format: 'API' | 'UI'): string {
 	}
 }
 
+function getRandomFieldEntryValue(
```

Comment:
Hey @G-Sa, could you add the object's prefix here, so `getRandomObjectFieldEntryValue`?